sympy/stats/joint_rv_types.py

```diff
@@ -58,3 +58,152 @@ def marginal_distribution(self, indices, sym):
         return Lambda(sym, S(1)/sqrt((2*pi)**(len(_mu))*det(_sigma))*exp(
             -S(1)/2*(_mu - sym).transpose()*(_sigma**(-1)*\
                 (_mu - sym)))[0])
+
+#-------------------------------------------------------------------------------
+# Multivariate Laplace distribution ---------------------------------------------------------
+
+class MultivariateLaplaceDistribution(JointDistribution):
+    _argnames = ['mu', 'sigma']
+    is_Continuous=True
+
+    @property
+    def set(self):
+        k = len(self.mu)
+        return S.Reals**k
+
+    def check(self, mu, sigma):
+        mu, sigma = Matrix([mu]), Matrix(sigma)
+        _value_check(len(mu) == len(sigma.col(0)),
+            "Size of the mean vector and covariance matrix are incorrect.")
+        #check if covariance matrix is positive definite or not.
+        _value_check(all([i > 0 for i in sigma.eigenvals().keys()]),
+            "The covariance matrix must be positive definite. ")
+
+    def pdf(self, *args):
+        from sympy.functions.special.bessel import besselk
+        mu, sigma = Matrix(self.mu), Matrix(self.sigma)
+        k = len(mu)
+        sigma_inv = sigma**(-1)
+        args = Matrix(args)
+        v = 1 - S(k)/2
+        return S(2)/((2*pi)**(S(k)/2)*sqrt(det(sigma))) \
+        *((args.transpose()*sigma_inv*args)[0]/(2 + (mu.transpose()*sigma_inv*mu)[0])) \
```

Comment:
The returned expression would be more readable (and more efficient) if `args.transpose()` and `mu.transpose()` were precomputed and assigned to local variables. Even `(mu.transpose()*sigma_inv*mu)[0]` could be precomputed.